internal/vlan/create.go

```diff
@@ -45,28 +46,30 @@ func (c *Client) Create() *cobra.Command {
 
 		RunE: func(cmd *cobra.Command, args []string) error {
 			cmd.SilenceUsage = true
-
-			req := &packngo.VirtualNetworkCreateRequest{
-				ProjectID: projectID,
-				Metro:     metro,
-				Facility:  facility,
-				VXLAN:     vxlan,
+			virtualNetworkCreateInput := metal.NewVirtualNetworkCreateInput()
+			if metro != "" {
+				virtualNetworkCreateInput.SetMetro(metro)
+			}
+			if facility != "" {
+				virtualNetworkCreateInput.SetFacility(facility)
 			}
+			virtualNetworkCreateInput.SetVxlan(int32(vxlan))
+
 			if description != "" {
-				req.Description = description
+				virtualNetworkCreateInput.Description = &description
 			}
 
-			n, _, err := c.Service.Create(req)
+			n, _, err := c.Service.CreateVirtualNetwork(context.Background(), projectID).VirtualNetworkCreateInput(*virtualNetworkCreateInput).Include(nil).Exclude(nil).Execute()
```

Comment:
There's no need to call `Include` or `Exclude` with a hard-coded `nil`; passing in `nil` is equivalent to not calling the method at all.

```suggestion
			n, _, err := c.Service.CreateVirtualNetwork(context.Background(), projectID).VirtualNetworkCreateInput(*virtualNetworkCreateInput).Execute()
```